### ISSUE TITLE
Add more intelligence ISOBAR Farm interaction

### ISSUE THREAD

**Issue by lmhare** [JeffersonLab/PyPWA#91]
While running the simulation tutorial an error came up that stated the following:

```
Traceback (most recent call last):
  File "/lustre/expphy/work/hallb/clasg12/lmhare/phi/phi_simulation/scripts/massBinner.py", line 196, in <module>
    mB = massBinner(indir=sys.argv[1],bindir=sys.argv[2],gfile=sys.argv[3],verb=sys.argv[4])
  File "/lustre/expphy/work/hallb/clasg12/lmhare/phi/phi_simulation/scripts/massBinner.py", line 47, in __init__
    self.gampList=self.gampT.translate(os.path.join(self.indir,self.nfile))
  File "/lustre/expphy/work/hallb/clasg12/lmhare/phi/phi_simulation/pythonPWA/pythonPWA/fileHandlers/gampTranslator.py", line 81, in translate
    self.readFile()        
  File "/lustre/expphy/work/hallb/clasg12/lmhare/phi/phi_simulation/pythonPWA/pythonPWA/fileHandlers/gampTranslator.py", line 60, in readFile
    self.events[n,i,1]= particle[1]
IndexError: list index out of range
```

From there while checking the directory structure there were no mass bins created from the simulation tutorial. In addition, the mass binner did not run during the initial instillation. As seem below, no bins were created. 

```
/lustre/expphy/work/hallb/clasg12/lmhare/phi/phi_simulation
ifarm1401> ls
events.pf  GUI  keyfiles  pythonPWA  raw_events.gamp  scripts  simulation
ifarm1401> ls -al
total 813303
drwxr-sr-x 7 lmhare clasg12      4096 Jul 14 10:52 .
drwxr-sr-x 8 lmhare clasg12      4096 Jul 14 10:05 ..
-rw-r--r-- 1 lmhare clasg12  35200000 Jul 14 10:07 events.pf
drwxr-sr-x 3 lmhare clasg12      4096 Jul 14 10:52 GUI
drwxr-sr-x 2 lmhare clasg12      4096 Jul 14 10:52 keyfiles
drwxr-sr-x 8 lmhare clasg12      4096 Jul 14 10:10 pythonPWA
-rw-r--r-- 1 lmhare clasg12 797474816 Jul 14 10:08 raw_events.gamp
drwxr-sr-x 3 lmhare clasg12      4096 Jul 14 10:52 scripts
drwxr-sr-x 3 lmhare clasg12      4096 Jul 14 10:52 simulation
ifarm1401> ls -al simulation/
total 12
drwxr-sr-x 3 lmhare clasg12 4096 Jul 14 10:52 .
drwxr-sr-x 7 lmhare clasg12 4096 Jul 14 10:52 ..
drwxr-sr-x 2 lmhare clasg12 4096 Jul 14 10:52 overflow
```

**Comment by Markjonestx** [comment]
This issue appears to be caused by a timeout issue or a failure on ISOBAR's part when pushing the binning to the Farm. Ultimately, the ISOBAR needs to track which jobs were successful and which were not and alert the user to any issues.

**Comment by Markjonestx** [comment]
Merged into #108
